6.3_a_long_cat_is_long.py

```diff
@@ -0,0 +1,22 @@
+def count_words(text):
+    """
+    The function accepts text and returns a dictionary where the key is a word and the value is the length of the word.
+    using generator expression to clean the text from signs that are not letters
+    and using dictionary comprehension to generate the dictionary
+    :param text: text
+    :return: dictionary_text
+    """
+    temp_text = (w for w in text if w.isalpha() or w == " ")  # generator expression
```

Comment:
replace 'w' Variable names - https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing

Reply:
I changed w to char (and temp_text to generator_text)